packages/website/docusaurus.config.js

```diff
@@ -84,34 +84,34 @@ const config = {
       },
       footer: {
         style: "dark",
-        links: [
-          {
-            title: "Docs",
-            items: [
-              {
-                label: "Introduction",
-                to: "/",
-              },
-              {
-                label: "Language basics",
-                to: "/language-basics/overview",
-              },
-            ],
-          },
-          {
-            title: "Community & Support",
-            items: [
-              {
-                label: "Stack Overflow",
-                href: "https://stackoverflow.microsoft.com/search?q=cadl",
-              },
-              {
-                label: "Microsoft Teams Channel",
-                href: "http://aka.ms/cadl/discussions",
-              },
-            ],
-          },
-        ],
+        // links: [
```

Comment:
Do we not want this? should then maybe just remove?

Reply:
We probably would need something like this either point to repo issue reporting. Let's get this in. Mark will be sharing out for his customer visit next week.